generate-moz-grammars/generate-mozcpp.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script updates the mozcpp grammar automatically.
+#
+# Usage: ./generate-moz-grammars/generate-mozcpp.sh
+
+# FIXME we need to remove this line once we are going to use
+# the tree-sitter-cpp bindings
+# Get the tree-sitter-cpp submodule version
+TS_CPP_VERSION=`git submodule status tree-sitter-cpp | awk '{ print $1 }'`
+
+# Enter the mozcpp directory
+pushd tree-sitter-mozcpp
+
+# Download tree-sitter-cpp
+git clone https://github.com/tree-sitter/tree-sitter-cpp.git
```

Comment:
Nit: it'd be nice to use shallow clones so we directly clone the right revision and only the files at that revision